cfgov/jinja2/v1/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -0,0 +1,524 @@
+        'url': 'http://www.consumerfinance.gov/orgmember/operations/administrative-operations/',
+        'title': 'Administrative Operations',
+        '_id': 'administrative-operations',
+        'order': 10
+        }, {
+        'relative_url': '/orgmember/operations/chief-financial-office-cfo/',
+        'titles': ['Assistant Director'],
+        'id': 36711,
+        'category': 'Divisions',
+        'parent': 36707,
+        'slug': 'chief-financial-office-cfo',
+        'comment_status': 'closed',
+        'title_plain': 'Chief Financial Officer (CFO)',
+        'name': 'Stephen Agostini',
+        'url': 'http://www.consumerfinance.gov/orgmember/operations/chief-financial-office-cfo/',
+        'title': 'Chief Financial Officer (CFO)',
+        '_id': 'chief-financial-office-cfo',
+        'order': 20
+        }, {
+        'relative_url': '/orgmember/operations/human-capital-chief-human-capital-officer-chco/',
+        'titles': ['Chief Human Capital Officer (CHCO)', 'Assistant Director'],
+        'id': 36713,
+        'category': 'Divisions',
+        'parent': 36707,
+        'slug': 'human-capital-chief-human-capital-officer-chco',
+        'title_plain': 'Human Capital',
+        'name': 'Jeffrey Sumberg',
+        'url': 'http://www.consumerfinance.gov/orgmember/operations/human-capital-chief-human-capital-officer-chco/',
+        'title': 'Human Capital',
+        '_id': 'human-capital-chief-human-capital-officer-chco',
+        'order': 30
+        }, {
+        'relative_url': '/orgmember/operations/technology-and-innovation/',
+        'titles': ['Chief Information Officer (COO)', 'Assistant Director'],
+        'id': 36715,
+        'category': 'Divisions',
+        'parent': 36707,
+        'slug': 'technology-and-innovation',
+        'title_plain': 'Technology and Innovation',
+        'name': 'Ashwin Vasan',
+        'url': 'http://www.consumerfinance.gov/orgmember/operations/technology-and-innovation/',
+        'title': 'Technology and Innovation',
+        '_id': 'technology-and-innovation',
+        'order': 40
+        }, {
+        'relative_url': '/orgmember/operations/office-of-consumer-response/',
+        'titles': ['Assistant Director<sup>*</sup>'],
+        'id': 36717,
+        'category': 'Divisions',
+        'parent': 36707,
+        'slug': 'office-of-consumer-response',
+        'title_plain': 'Office of Consumer Response',
+        'name': 'Christopher Johnson',
+        'url': 'http://www.consumerfinance.gov/orgmember/operations/office-of-consumer-response/',
+        'title': 'Office of Consumer Response',
+        '_id': 'office-of-consumer-response',
+        'order': 50
+        }, {
+        'relative_url': '/orgmember/operations/chief-procurement-officer-cpo/',
+        'titles': ['Assistant Director'],
+        'id': 36719,
+        'category': 'Divisions',
+        'parent': 36707,
+        'slug': 'chief-procurement-officer-cpo',
+        'title_plain': 'Chief Procurement Officer (CPO)',
```

Comment:
Why `title_plain` and `title` with the same value? 

Also, is `titles` really just a plural of the value in `title`, or could those be named more distinctly to convey when one vs the other is used?